### ISSUE TITLE
Dist for linux aarch64

### ISSUE THREAD

**Issue by niyaznigmatullin** [Z3Prover/z3#6835]
Any chance the library for aarch64 Linux is included in the releases?

**Comment by NikolajBjorner** [comment]
Can you check if the bits in the nightly work?

https://github.com/Z3Prover/z3/releases/tag/Nightly

**Comment by niyaznigmatullin** [comment]
Seems like it's still for `x86-64`

```
# file libz3.so
libz3.so: ELF 64-bit LSB shared object, x86-64, version 1 (GNU/Linux), dynamically linked, BuildID[sha1]=348aef9ebf41d859d6984ba04e86ac0c59027157, not stripped
```

**Comment by NikolajBjorner** [comment]
try now

**Comment by niyaznigmatullin** [comment]
Seems working, thanks!

**Comment by no-preserve-root** [comment]
It seems the arm64-glibc-2.35 distribution is missing from the 4.12.3 release; would it be possible to add that?

**Comment by robby-phd** [comment]
The recently released [z3-4.12.4-arm64-glibc-2.35.zip](https://github.com/Z3Prover/z3/releases/download/z3-4.12.4/z3-4.12.4-arm64-glibc-2.35.zip ) is somehow for x86_64, e.g.,

```sh
wget -q https://github.com/Z3Prover/z3/releases/download/z3-4.12.4/z3-4.12.4-arm64-glibc-2.35.zip && unzip -q z3-4.12.4-arm64-glibc-2.35.zip && file z3-4.12.4-arm64-glibc-2.35/bin/z3
```

produces:

```
z3-4.12.4-arm64-glibc-2.35/bin/z3: ELF 64-bit LSB pie executable, x86-64, version 1 (GNU/Linux), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, BuildID[sha1]=6fe4b481f83ac009b1089362792b976157caa239, for GNU/Linux 3.2.0, not stripped
```

**Comment by robby-phd** [comment]
@NikolajBjorner  The recently released [z3-4.12.5-arm64-glibc-2.35.zip](https://github.com/Z3Prover/z3/releases/download/z3-4.12.5/z3-4.12.5-arm64-glibc-2.35.zip) is also for x86_64, e.g.,

```sh
wget -q https://github.com/Z3Prover/z3/releases/download/z3-4.12.5/z3-4.12.5-arm64-glibc-2.35.zip && unzip -q z3-4.12.5-arm64-glibc-2.35.zip && file z3-4.12.5-arm64-glibc-2.35/bin/z3
```

produces:

```
z3-4.12.5-arm64-glibc-2.35/bin/z3: ELF 64-bit LSB pie executable, x86-64, version 1 (GNU/Linux), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, BuildID[sha1]=4a19713095c918e63ef87323d77efc3693c55965, for GNU/Linux 3.2.0, not stripped
```

**Comment by shaoweilin** [comment]
If anyone needs it, the z3-4.12.5-arm64-glibc-2.39 release can be found on this fork: https://github.com/Beneficial-AI-Foundation/z3/releases/tag/z3-4.12.5
